src/observers/OmniSharpConfigChangedObserver.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { vscode } from '../vscodeAdapter';
+import 'rxjs/add/operator/debounceTime';
+import { Options } from '../omnisharp/options';
+import showWarningMessage from './ShowWarningMessage';
+
+let options: Options;
+
+export async function WorkspaceConfigurationChangedHandle(vscode: vscode, newOptions: Options) {
+    if (options != null) {
```

Comment:
Instead of trying to store the last option you see, I would hook this up to a "pairiwise" so you can compare each value to the previous one https://www.learnrxjs.io/operators/combination/pairwise.html